src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,96 @@
+drop table if exists favorite
+;
+
+drop table if exists line
+;
+
+drop table if exists member
+;
+
+drop table if exists section
+;
+
+drop table if exists station
+;
+
+create table favorite
```

Comment:
```suggestion
create if not exists table favorite
```

drop하지 않고 위와 같이 작성하는 방법도 있습니다 :)